src/components/base/text-area/index.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { forwardRef } from "react";
+import propTypes from "prop-types";
+import { TEXTAREA_DEFAULT_ROW } from "utils/const";
+
+const TextArea = forwardRef(function TextArea(props, ref) {
```

Comment:
component doesn't have a default state, consider introducing a `value` prop and it should be exposed via `onChange` prop.